Applications/Express.md

```diff
@@ -0,0 +1,92 @@
+## Express
+
+The express project is applying to become an incubating
+[Top Level Project of the Node.js Foundation][].
+
+### About Express, User base, Community and Ecosystem
+
+Express is one of the most popular web server frameworks available for Node.js.
+For many Node.js developers, express is their primary interface with Node.js
+on the server.
+
+Express has consistently been among the top packages installed via npm, with
+nearly 4.5 million downloads in just the past month.
+
+It is important to recognize that Express is not the only web framework in the
+Node.js ecosystem and that by incubating Express, neither the Node.js
+Foundation or Node.js core would be "picking a winner" or endorsing Express
+over any other overlapping or similar project.
+
+### Scope 
+
+The express project is composed of a collection of separate modules and 
+documentation currently spread out over multiple GitHub organizations. This top 
+level project would consist initially of the following GitHub Organizations and
+repositories within:
+
+* jshttp
+* expressjs
+* pillarjs
+
+The following projects will move into one of these organizations.
+
+* other
+  * strongloop/express
+  * component/escape-html
+  * dougwilson/depd
+  * visionmedia/bytes.js
+  * crypto-utils/random-bytes
+  * crypto-utils/uid-safe
+
+In addition, strongloop will contribute the basic express API documentation 
+included in the strongloop/expressjs.com to the strongloop/express repository.
+
+Many repositories in these organizations may be moved out or removed entirely
+during incubation at the discretion of the Express TSC. Prior to graduating
+from incubation a full list of repositories will need to be provided to the TSC.
```

Comment:
TSC is appropriate here. Those repositories will be selected by the TC but need to be included in the charter that is reviewed by and approved by the TSC